nornir/plugins/functions/text/__init__.py

```diff
@@ -115,7 +161,8 @@ def print_result(
     vars: List[str] = None,
     failed: bool = False,
     severity_level: int = logging.INFO,
-) -> None:
+    return_output: bool = False,
```

Comment:
instead of using a bool for this and then the global `OUT_BUFFER` I'd suggest the user passes the `StringIO`. Something like:

```
dest: Optional[StringIO] = None
```